Configurations/windows-makefile.tmpl

```diff
@@ -410,19 +414,14 @@ all: build_sw build_docs
 test: tests
 {- dependmagic('tests'); -}: build_programs_nodep build_modules_nodep copy-utils
 	@{- output_off() if $disabled{tests}; "" -}
-	set SRCTOP=$(SRCDIR)
-	set BLDTOP=$(BLDDIR)
-	set PERL=$(PERL)
-	set FIPSKEY=$(FIPSKEY)
-	"$(PERL)" "$(SRCDIR)\test\run_tests.pl" $(TESTS)
+	cmd /C "set "SRCTOP=$(SRCDIR)" & set "BLDTOP=$(BLDDIR)" & set "PERL=$(PERL)" & set "FIPSKEY=$(FIPSKEY)" & "$(PERL)" "$(SRCDIR)\test\run_tests.pl" $(TESTS)"
```

Comment:
The quotes inside quotes don't look right...

Reply:
I've verified it working like this. [MSDN](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/cmd#remarks) says:

>If you specify **/c** or **/k**, **cmd** processes, the remainder of _string_, and the quotation marks are preserved only if all of the following conditions are met:
>...
>If the previous conditions aren't met, _string_ is processed by examining the first character to verify whether it is an opening quotation mark. If the first character is an opening quotation mark, it is stripped along with the closing quotation mark. Any text following the closing quotation marks is preserved.

Only the first and last quotes are removed, and any inner quotes remain in the command-line passed on to `cmd.exe`.

Reply:
No need to change, that reminder of how command lines are treated on Windows is sufficient.

I'm always thrown off by the implicit nature of what quotes remain or not.  It's not clear to my eyes which quote is the "closing quotation mark", but then, I'm culturally much closer to Unix and VMS, where the treatment of quotes is done by the shell, and where the intent is much more explicit.